include/benchmark/benchmark.h

```diff
@@ -572,35 +572,38 @@ class State {
     return (max_iterations - total_iterations_ + batch_leftover_);
   }
 
- private:
-  bool started_;
-  bool finished_;
+private: // items we expect on the first cache line (ie 64 bytes of the struct)
+  bool started_ : 1;
```

Comment:
how about not doing this yet, but adding a "TODO: make these bitfields if required".

reasoning: on some platforms (at least this used to be true), bitfield accesses required quite a bit more ASM than bool accesses. Ie, the bitfield work might negate the work to get these in the cache line in the first place.

Reply:
Sounds reasonable to me.

Perhaps we could still attack the low hanging fruit and enable it on x86?